crates/uv/src/commands/python/install.rs

```diff
@@ -665,13 +711,20 @@ pub(crate) async fn install(
                 )?;
             }
         }
+
+        if fatal {
+            return Ok(ExitStatus::Success);
+        }
```

Comment:
Oops, this is correct but per the variable name should be inverted. Fixed in https://github.com/astral-sh/uv/pull/14614